### PR TITLE
Space count bug

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class DemoWidget extends Component {
         }
       },
       enableSpaceListFilter: this.state.enableSpaceListFilter,
-      spaceLoadCount: this.state.spaceLoadCount,
+      spaceLoadCount: Number(this.state.spaceLoadCount),
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js]: I like this change. Makes things much cleaner!

File: packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js [hunk 2]
```diff
@@ -475,7 +469,7 @@ class DemoWidget extends Component {
                   inputSize="small-12"
                   type="number"
                   onChange={this.handleRecentSpaceLoadCountChange}
-                  value={this.state.spaceLoadCount}
+                  placeholder={this.state.spaceLoadCount}
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js]: What was the benefit of changing this to `placeholder` from `value`?

File: packages/node_modules/@ciscospark/widget-recents/src/enhancers/setup.js [hunk 3]
```diff
@@ -181,14 +181,15 @@ function getInitialSpaces(props) {
...
-    widgetStatus
+    widgetStatus,
+    spaceLoadCount
   } = props;
 
   if (!widgetStatus.isFetchingAllSpaces
     && !widgetStatus.hasFetchedAllSpaces) {
     props.updateWidgetStatus({isFetchingAllSpaces: true});
     props.fetchSpaces(sparkInstance, {
-      limit: 250
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-recents/src/enhancers/setup.js]: Good catch!

File: packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js (hunk 2 above)
Comment (akoushke) [reply packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js]: addressed in the space.

File: packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js [hunk 4]
```diff
@@ -225,13 +225,7 @@ class DemoWidget extends Component {
...
   @autobind
   handleRecentSpaceLoadCountChange(e) {
-    let parsed = parseInt(e.target.value, 10);
-
-    if (Number.isNaN(parsed)) {
-      parsed = 25;
-    }
-
-    this.setState({spaceLoadCount: parsed});
+    this.setState({spaceLoadCount: e.target.value ? e.target.value : ''});
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js]: Works great!

**Comment by adamweeks** [comment]
Code was merged via gauntlet but this branch was not updated, so manually closing.

**Pull request closed without merging**